lib-ee/emqx_ee_bridge/src/emqx_ee_bridge.erl

```diff
@@ -82,14 +83,6 @@ resource_type(clickhouse) -> emqx_ee_connector_clickhouse.
 
 fields(bridges) ->
     [
-        {kafka,
```

Comment:
Q: Should it be deprecated instead of straight deletion?

Reply:
renamed to `kafka_producer` ?
We'll need to add a `aliases` feature in hocon schema so we can more freely rename rename fields without writing a lot of converters for backward compatibility

Reply:
Yes, it was renamed to `kafka_producer`, and its schema was also changed.  That was because it had a structure that would be more suitable if the bridge were to behave bidirectionally, like the MQTT bridge.  But that doesn't work well for the Kafka Producer / Consumer duo.

Reply:
we'll have to keep backward compatibility for this change.